extensions/office-quotes/src/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { ActionPanel, Action, List, Icon, Toast, showToast } from "@raycast/api";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { DescriptionPane } from "./DescriptionPane";
+import { shuffle } from "./shuffle";
+
+export default function main() {
+  const [loading, setLoading] = useState(true);
+  const [quotes, setQuotes] = useState([]);
+
+  useEffect(() => {
+    if (quotes.length === 0) {
+      axios
+        .get("https://officeapi.dev/api/quotes/")
+        .then((res) => {
+          const shuffledQuotes: any = shuffle(res.data.data);
+          setQuotes(shuffledQuotes);
+        })
+        .then(() => setLoading(false))
+        .catch((err) => {
+          showToast(Toast.Style.Failure, "Failed to load quotes", err.message).then((err) => console.log(err));
+        });
+    }
+  }, []);
+
+  return (
+    <List isLoading={loading}>
+      {quotes?.length > 0 &&
+        quotes.map((quote, index) => (
+          <List.Item
+            title={quote["content"]}
+            key={quote["content"]}
+            accessories={[
```

Comment:
```
accessories={[
    { text: `${quote["character"]["firstname"]} ${quote["character"]["lastname"]}` },
    { icon: Icon.Person },
]}
```

I think you can do this, so Person icons are aligned

Reply:
Good call, thank you! I will make the adjustment today.